code/src/index.css

```diff
@@ -5,9 +5,134 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.header {
+  font-family: Helvetica;
+  color: #ffffff;
+  line-height: 10px;
+  margin-left: 10px;
+  font-size: 18px;
+}
+
+.song-title {
+  font-family: Helvetica;
+  line-height: 10px;
```

Comment:
Line height needs to be increased, it does not look great when the title is longer and takes up two rows.